.azure-pipelines/security-tools.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   steps:
   - checkout: self
     fetchTags: false
-  - template: util/get-github-pat-steps.yml
```

Comment:
Want to double check we don't need to delete this yml itself because it's fine to use it in internal pipelines, correct?

Reply:
I want to delete it. Code Gen pipeline is using it. Let's delete it and update code gen pipeline in another PR.